scripts/apps/workspace/views/workspace-dropdown.html

```diff
@@ -27,7 +26,7 @@ <h1 id="workspace-heading" class="a11y-only"><span translate>Workspace:</span> {
             </button>
         </li>
         <li>
-            <button class="action-btn" ng-click="createWorkspace()">
+            <button class="action-btn" ng-click="createWorkspace()" data-test-id="create-workspace">
```

Comment:
drop data-test-id and get the element by button text:
`await page.getByRole('button', {name: 'create new workspace'}).click();`